app/views/users/index.html.erb

```diff
@@ -1,2 +1,14 @@
-<h1>Here you will find a list of all the users</h1>
+<div class="users-container black-border">
+  <% @users.each do |user| %>
+    <div class="user-container">
+      <div class="user-img black-border">
+        <%= image_tag("icon.png") %>
+      </div>
+      <div class="user-overview black-border">
+        <h2><%= link_to user.name, user_path(user), class: "link-style" %></h2>
+        <span>Number of posts: <%= user.posts_counter || 0 %></span>
+      </div>
+    </div>
+  <% end %>
+</div> 
```

Comment:
- [ ] **[OPTIONAL]** I would suggest creating partial `app/views/users/_user.html.erb` for rendering the user's details since it's being used in several pages 💯. You can then render the partial where you need to render the user's details above. This will improve your knowledge of rails partial and also keep your codebase DRY. Read more about partials [here ](https://riptutorial.com/ruby-on-rails/example/2875/partials)